plugins/core/gs-desktop-common.c

```diff
@@ -271,9 +275,8 @@ static const GsDesktopMap map_reference[] = {
 	{ "lifestyle",		NC_("Menu of Reference", "Lifestyle"),
 					{ "Reference::Lifestyle",
 					  NULL} },
-	{ "news",		NC_("Menu of Communication & News", "News"),
-					{ "Network::Feed",
-					  "Network::News",
+	{ "news",		NC_("Menu of Reference News", "News"),
```

Comment:
@joaquimrocha I think this should just say "Menu of Reference", for consistency with all the other subcategories.